components/match2/wikis/arenaofvalor/match_summary.lua

```diff
@@ -162,7 +162,7 @@ function CustomMatchSummary.createBody(match)
 	if not Table.isEmpty(championBanData) then
 		local championBan = ChampionBan()
 
-		for gameIndex, banData in ipairs(championBanData) do
+		for gameIndex, banData in pairs(championBanData) do
```

Comment:
pairs wouldn't guarantee that they come in the right order

Reply:
right...